hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java

```diff
@@ -1117,7 +1117,7 @@ public short getMinReplicationToBeInMaintenance() {
     return minReplicationToBeInMaintenance;
   }
 
-  private short getMinMaintenanceStorageNum(BlockInfo block) {
+  public short getMinMaintenanceStorageNum(BlockInfo block) {
```

Comment:
It can be package private.
```suggestion
 short getMinMaintenanceStorageNum(BlockInfo block) {
```